tests/test_simple_nlp.py

```diff
@@ -269,7 +269,7 @@ def test_simple_nlp_mlm_loss_gen_token_batch(tiny_bert_tokenizer, device):
         model=model,
         seed=42,
         train_dataloader=data_spec,
-        max_duration='2ep',
+        max_duration='3ep',
```

Comment:
Using 2 epochs makes the `cross_entropy` and `token_cross_entropy` the same for torch 2.7 but since even in earlier torch versions, these values are really similar, I added an extra epoch to exaggerate this loss slightly to make the tests pass.

Reply:
A little worried about the loss values changing post torch bump -- not expecting changes here. Are they the same loss as before? Or different values all together? (I'm seeing `10.4344` on torch 2.6)

Reply:
Maybe reasonable due to cuda bump...

Reply:
Issue happens on non-CUDA bump as well :( (downgraded to CUDA 12.6.3)

Reply:
Will reopen new threads

Reply:
Could you try exaggerating this by making the dtms smaller instead of increasing epochs?

Reply:
Yeah that worked